App/index.js

```diff
@@ -0,0 +1,21 @@
+function unhidePlaylist(id){
+    wantedId = null;
+    Playlist[0].wantedId = 0;
+    Playlist[1].wantedId = 1;
+    Playlist[2].wantedId = 2;
+    var selectedPlaylist = null;
+    for(var i = 0; i <= Playlist.length; i++){
```

Comment:
Playlist is the constructor function, use another name like  playlistCollection.